tests/network-tests/src/Scenario.ts

```diff
@@ -48,8 +49,11 @@ function writeOutput(api: Api, miniSecret: string) {
 }
 
 export async function scenario(label: string, scene: (props: ScenarioProps) => Promise<void>): Promise<void> {
-  // Load env variables
-  config()
+  // Load env variables - test framework specific
+  console.log(config({ path: path.join(__dirname, '../.env') }))
```

Comment:
TODO: bad idea to log env variables in-case there are sensitive secrets in here (left over from debugging)

Reply:
fixed in 74909c991bb48a3114f893fba6da9a22943a7d74